internal/auth0/branding/resource_theme.go

```diff
@@ -35,60 +35,69 @@ func NewThemeResource() *schema.Resource {
 			"borders": {
 				Type:     schema.TypeList,
 				MaxItems: 1,
-				Required: true,
+				Optional: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"buttons_style": {
 							Type:         schema.TypeString,
 							ValidateFunc: validation.StringInSlice([]string{"pill", "rounded", "sharp"}, false),
-							Required:     true,
+							Optional:     true,
+							Default:      "rounded",
 							Description:  "Buttons style. Available options: `pill`, `rounded`, `sharp`.",
```

Comment:
Could we please update as well all the descriptions to mention what the default value will be 🙏🏻 ? e.g.
```suggestion
							Description:  "Buttons style. Available options: `pill`, `rounded`, `sharp`. Defaults to `rounded`. ",
```